test/book/cornerstones/groups/auxiliary_code/main.jl

```diff
@@ -1,5 +1,5 @@
 import Pkg
-Pkg.add(name="GenericCharacterTables", version=v"0.2"; io=devnull)
+Pkg.add(name="GenericCharacterTables", version="0.3"; io=devnull)
```

Comment:
This will choose the latest 0.3.x release of GCT.
Other possible options are:
1. Fix a specific version
```suggestion
Pkg.add(name="GenericCharacterTables", version=v"0.3.1"; io=devnull)
```
2. Use the latest version (even breaking)
```suggestion
Pkg.add(name="GenericCharacterTables"; io=devnull)
```

Reply:
(all of these of course wait for @fingolfin to release a 0.3.1 version)

Reply:
If/once https://github.com/oscar-system/GenericCharacterTables.jl/pull/90 is merged and released, this instead should be changed to
```suggestion
Pkg.add(name="GenericCharacterTables", version="0.2"; io=devnull)
```

(@benlorenz it is unfortunately not possible to to more involved compat strings here. The only possibility is to provide a string missing the patch version (or missing patch and minor), which makes julia choose any of them. As long as you don't have any version installed and have a rather new general registry (as both is the case in CI), this should then choose the newest applicable patch version.
The only other possibility would be to fix a concrete version (like 0.2.2) here, and need to bump it with every release of GCT.)